src/main/java/net/sf/jabref/logic/integrity/IntegrityCheck.java

```diff
@@ -321,6 +324,12 @@ private BracketChecker(String field) {
                 return Collections.singletonList(new IntegrityMessage(Localization.lang("should contain a four digit number"), entry, FieldName.YEAR));
             }
 
+            if (!ENDS_WITH_FOUR_DIGIT.test(value.get().replaceAll(regex, ""))) {
+                return Collections.singletonList(new IntegrityMessage(
+                        Localization.lang("last four nonpunctuation characters should be numerals"), entry,
```

Comment:
Most messages do actually start with a lower case character right now so
for consistency it is currently correct. Although I think that conceptually
it should be upper case.